WcaOnRails/config/locales/en.yml

```diff
@@ -1459,6 +1459,12 @@ en:
         search: "Name or city"
         #context: the "recent" button
         recent: "Last %{count} days"
+        #context: the registration status icon
+        registration:
+          opens_in: "Registration will open in %{duration}"
+          closed: "Registration has closed"
```

Comment:
Change to `Registration has closed. Competition starts in %{duration}`?
Tips the hat to the old hourglass which had a "competition starts in %duration" tooltip.